assets/js/plugin-check-admin.js

```diff
@@ -283,7 +283,7 @@
 	 * @param {Object} warnings The file warnings.
 	 */
 	function renderFileResults( file, errors, warnings ) {
-		const index = Date.now();
+		const index = parseInt( Math.random() * 100000, 10 );
```

Comment:
I have used [uuid](https://www.npmjs.com/package/uuid) in the past. But adding this library might overkill. 

This might be another option. 

```
const uid = function(){
    return Date.now().toString(36) + Math.random().toString(36).substr(2);
}
``` 
From https://stackoverflow.com/a/53116778

Reply:
Thanks! Addressed in https://github.com/10up/plugin-check/pull/155/commits/b8a9e01f60b884aaf4ed51ae629ea30ecd68afb1